examples/route_k8s_resource/README.md

```diff
@@ -99,22 +99,22 @@ cat <<EOS |kubectl apply -f -
 apiVersion: apps.openshift.io/v1alpha1
 kind: ServiceBindingRequest
 
-metadata: 
+metadata:
   name: binding-request
   namespace: service-binding-demo
 
-spec: 
-  applicationSelector: 
-    group: ""
+spec:
+  applicationSelector:
+    group: apps
```

Comment:
@noelo if I understand correctly, this line is only real change in PR? The rest of changes are whitespace formatting?